docs/source/getting-started/quickstart.rst

```diff
@@ -165,8 +165,26 @@ Stop/terminate a cluster
 =========================
 
 When you are done, run :code:`sky stop mycluster` to stop the cluster. To
-terminate a cluster instead, run :code:`sky down mycluster`.  Find more commands that
-manage the lifecycle of clusters :ref:`here <interactive-nodes>`.
+terminate a cluster instead, run :code:`sky down mycluster`.
+
+Find more commands that manage the lifecycle of clusters in the :ref:`CLI reference <cli>`.
+
+Scaling out
+=========================
+
+So far, we have used the `serverful` CLIs of SkyPilot, where the notion of clusters is central.
+This is useful for development since it is convenient to log into a cluster and perform debugging.
+
+When you are ready to scale out (e.g., run 10s or 100s of jobs), SkyPilot supports two options:
+
+- Queue jobs on a cluster (see :ref:`Job Queue <job-queue>`); or
+- Use :ref:`Managed Spot Jobs <spot-jobs>`, which is a `serverless` style API where clusters are not directly exposed.
+
+Managed spot jobs run on much cheaper spot instances, with automatic preemption recovery. Try it out with:
```

Comment:
preemption is a systems-y word, failure might be better understood
```suggestion
Managed spot jobs run on much cheaper spot instances, with automatic recovery from failures. Try it out with:
```

Reply:
I think it's ok to say preemptions; people who know "spot" should know about that word. "Failures" can mean user code failures which we cannot recover from.